src/OpenLoco/src/Paint/PaintStation.h

```diff
@@ -9,4 +12,11 @@ namespace OpenLoco::Paint
     struct PaintSession;
 
     void paintStation(PaintSession& session, const World::StationElement& elStation);
+
+    void paintAirport(PaintSession& session, const World::StationElement& elStation);
+    void paintDocks(PaintSession& session, const World::StationElement& elStation);
+    void paintRoadStation(PaintSession& session, const World::StationElement& elStation);
+    void paintTrainStation(PaintSession& session, const World::StationElement& elStation);
```

Comment:
With these functions going into their own unit files, I think it will be less confusing if they get their own header files as well.